src/org/usfirst/frc/team467/robot/TalonSpeedControllerGroup.java

```diff
@@ -94,8 +115,8 @@ public void disable() {
 			return;
 		}
 		leader.disable();
-		follower1.disable();
-		follower2.disable();
+		if(follower1 != null) follower1.disable();
+		if(follower2 != null) follower2.disable();
```

Comment:
I generally don't like these one-line if statements. It's ok this time, but next chance we get we need to fix these to be proper multi-line statements.